secret-service/pkg/backend/secretbackend_k8s_test.go

```diff
@@ -223,9 +222,25 @@ func TestDeleteK8sSecret(t *testing.T) {
 		ScopesRepository:       scopesRepository,
 	}
 
+	kubernetes.Fake.PrependReactor("list", "secrets", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
```

Comment:
If no special logic should be applied when a request for retrieving an object from the fake K8s service should be executed, the object can instead be passed to the `NewSimpleClientSet()` function - afterwards, it can be retrieved, updated and deleted without implementing a specific reactor for each request. Example:

```
kubernetes := k8sfake.NewSimpleClientset(&corev1.Secret{
		ObjectMeta: metav1.ObjectMeta{
			Name:      "my-secret",
			Namespace: "keptn_namespace",
			Labels:    map[string]string{"app.kubernetes.io/scope": "my-scope"},
		},
	},
		&rbacv1.Role{
			Rules: []rbacv1.PolicyRule{
				{
					Resources:     []string{"secrets"},
					ResourceNames: []string{"my-other-secret", "my-secret"},
				},
			},
		},
	)
```